plugin/acl/acl.go

```diff
@@ -73,7 +75,16 @@ RulesCheckLoop:
 			{
 				break RulesCheckLoop
 			}
+		case actionFilter:
+			{
+				m := new(dns.Msg)
+				m.SetRcode(r, dns.RcodeSuccess)
+				w.WriteMsg(m)
+				RequestBlockCount.WithLabelValues(metrics.WithServer(ctx), zone).Inc()
```

Comment:
It should have its own metric.

Reply:
I was on the fence about this before the initial submit, however you're likely right. Added.

Reply:
Yea, understand. Just my suggestion though.